rcsbapi/data/data_query.py

```diff
@@ -278,5 +268,8 @@ def reload(self):
 
         return ALL_STRUCTURES
 
-    def getAllIds(self, input_type):
-        return self.ALL_STRUCTURES[input_type]
+    def get_all_ids(self, input_type) -> List[str]:
+        if input_type in self.ALL_STRUCTURES:
+            return self.ALL_STRUCTURES[input_type]
+        else:
+            raise ValueError(f"ALL_STRUCTURES is not yet available for {input_type}")
```

Comment:
I'd suggest including the name `input_type` here too, so they know what parameter the error is referring to:

```suggestion
            raise ValueError(f"ALL_STRUCTURES is not yet available for input_type {input_type}")
```